packages/suite/src/middlewares/suite/__tests__/redirectMiddleware.test.ts

```diff
@@ -88,6 +88,7 @@ describe('redirectMiddleware', () => {
                 payload: {
                     device: getConnectDevice({ mode: 'initialize' }),
                     settings: { defaultWalletLoading: 'standard' },
+                    isViewOnlyByDefaultEnabled: true,
```

Comment:
nit: Just to clarify, this param will be changed also based on `auto eject wallets` value, right?

If so, wouldn’t it be better to call it `autoEjectWalletsEnabled` or something similar? Or are we already following the `desktop` implementation’s pattern?